src/frontend/check_annotation.nit

```diff
@@ -84,6 +84,7 @@ readonly
 writable
 autoinit
```

Comment:
`autoinit` can be used on normal methods (to indicate to use them in the auto-constructor) and on the class (to indicate the methods tu use the the auto-constructor, and their order)